whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/NrtmVersionInfoRepository.java

```diff
@@ -61,46 +61,20 @@ public Optional<NrtmVersionInfo> findLastVersion(final NrtmSource source) {
     }
 
     /**
-     * Creates a row in the version_information table for an initial snapshot.
+     * Creates a row in the version table for an initial snapshot.
      *
      * @param source The source which is being initialized
      * @param lastSerialId The last serialID from the Whois serials table which is in the snapshot
      * @return An initialized version object filled from the new row in the database
      */
-    public NrtmVersionInfo createInitialSnapshot(final NrtmSource source, final int lastSerialId) {
+    public NrtmVersionInfo createInitialVersion(final NrtmSource source, final int lastSerialId) {
         jdbcTemplate.update("INSERT INTO source (name) VALUES (?)", source.name());
         final long version = 1L;
-        final String sessionID = UUID.randomUUID().toString();
+        final String sessionID = NrtmFileUtil.sessionId();
```

Comment:
Does UUID need to be abstracted into `NrtmFileUtil`? Do we need to generate a UUID elsewhere?